### DISCUSSION TITLE
Making prediction on single unseen data

### DISCUSSION THREAD

**Discussion by qqfox** [stellargraph/stellargraph#1976] [Q&A]
Halo there,
I am performing the multi label classification task by using bert embedding to GCNs. Things works well except for the task of predictions. There are 2 problems:

Making prediction on unseen data. For example, use model to predict which label is for a new email.
    After trained and saved the model. Is it possible or any tutorial about how to make prediction on the new data (not in the dataset that used) based on the loaded model? And the data is either single items or multi items?
    Many thanks for any support

**Comment by asirico** [comment]
Is anyone else looking at these?

**Comment by davidshumway** [comment]
Any solution?
